libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Microsoft.AdaptiveDialog.schema

```diff
@@ -22,6 +22,20 @@
             "description": "Value that will be passed back to the parent dialog.",
             "default": "dialog.result"
         },
+        "dialogs": {
+            "type": "array",
+            "title": "Dialogs added to DialogSet.",
+            "dialog": {
```

Comment:
>dialog [](start = 13, length = 6)

This is not correct.  It should be items: { "oneOf": [...]}.  This doesn't mean anything.  You really should use dialog:merge to create something like the test schema and then open it in VS Code to create an instance of this $kind to make sure intellisense works as expected.

Reply:
I edited here. And test it as your description. The intellisense works now.